### PR TITLE
clean up tests to only use RFC 5737 and 5156 addresses

### PR DESCRIPTION
**Pull request by Megalonia**
Currently, we have existing tests which take in non-documentation IP addresses . This PR updates all test files such that IPs in test files are documentation addresses per RFC [5747](https://www.rfc-editor.org/rfc/rfc5737.html) and [5156](https://www.rfc-editor.org/rfc/rfc5156). This prevent tests from flaking/ booting users if they're on a given subnet.

### CODE REVIEW

File: tests/compat_ip.list [hunk 1]
```diff
@@ -1,6 +1,6 @@
 198.51.100.200
 198.51.100.1-198.51.100.100
 203.0.113.100/30
```
Comment (Megalonia) [inline tests/compat_ip.list:3]: should this be /24?

Comment (joewilliams) [reply tests/compat_ip.list:3]: A /30 is "inside" a /24 so it is safe to use.

**Review by joewilliams — CHANGES_REQUESTED** [review]
This is looking pretty good. Please make sure `make docker-ci` passes cleanly, please list the important details and rationale in the body of the PR.

Also, I think there are some addresses that still need to be changed in the go unit tests [here](https://github.com/ngrok/firewall_toolkit/blob/main/pkg/set/set_test.go#L504) but there might be other spots as well.

**Review by joewilliams — APPROVED** [review]
@Megalonia looks great, thank you! Please squash your commits before merging.

**Pull request merged**